admin-dev/themes/new-theme/scss/components/_alert.scss

```diff
@@ -2,4 +2,5 @@
   > a {
     padding: 0;
   }
+  min-height: 55px;
```

Comment:
Nitpick but please move it right after the selector line like

```
.alert {
  min-height: 55px;
```

for readability, this is a good practice